lms/djangoapps/grades/tests/test_signals.py

```diff
@@ -203,7 +206,7 @@ def set_up_course(self, enable_subsection_grades=True):
     def test_subsection_grade_updated_on_signal(self, default_store):
         with self.store.default_store(default_store):
             self.set_up_course()
-            with check_mongo_calls(2) and self.assertNumQueries(15):
+            with check_mongo_calls(2) and self.assertNumQueries(19):
```

Comment:
Really?  4 additional queries?
1 to check global flag.
1 to check all courses.
1 to check for the course
1 ??